mtags/src/main/scala-3/scala/meta/internal/pc/completions/Completions.scala

```diff
@@ -865,4 +934,114 @@ class Completions(
             else compareByRelevance(o1, o2)
       end compare
 
+  object MatchExtractor:
```

Comment:
Did you mean `MatchCaseCompletions`? How can I then provide them `completionPos`, `text` etc?

Reply:
Or we can move `CaseExtractor`, `CasePatternExtractor` and `TypedCasePatternExtractor` and leave only `MatchExtractor`, because they don't use anything besides `path`